File: data/data_original/at_coder/abc028/D/4588449.py
n,k = map(int,input().split())

ans = 0

#k???????
ans += (1/n)*(1/n)*(1/n)

#k?2?????
ans += 3*(1/n)*(1/n)*((n-1)/n)

#k?1??k???????1??k???????1?????
ans += 6*(1/n)*((k-1)/n)*((n-k)/n)

print(ans)

File: data/data_transpiled/at_coder/abc028/D/4588449.java
public static void print ( int n , int k ) {
  int ans = 0 ;
  ans += ( 1 / n ) * ( 1 / n ) * ( 1 / n ) ;
  ans += 3 * ( 1 / n ) * ( 1 / n ) * ( ( n - 1 ) / n ) ;
  ans += 6 * ( 1 / n ) * ( ( k - 1 ) / n ) * ( ( n - k ) / n ) ;
  System . out . println ( ans ) ;
}
